sass/_theme_mathjax.sass

```diff
@@ -3,3 +3,17 @@ span[id*='MathJax-Span']
 
 .math
```

Comment:
While you are at it, you might as well change this to

    .math p

See #686 for details.